index.html

```diff
@@ -0,0 +1,171 @@
+        </section>
+
+        <section class="intro-dogs">
+          <figure class="intro-img">
+            <img class="dogimage" src="images/duffy-checkout.jpg" alt="Picture of Duffy" onclick="dogInformation('Duffy', 'Terrier', '$123.45')">
+          </figure>
+          <h3>Duffy</h3>
+          <p><span class="dogs">Cost to Adopt:</span> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+          <button  class="dogs-button" type="submit" onclick="adoptionFee(123.45)">Adopt</button>
+        </section>
+
+        <section class="intro-dogs">
+          <figure class="intro-img">
+            <img class="dogimage" src="images/lucas-checkout.jpg" alt="Picture of Lucas" onclick="dogInformation('Lucas', 'Pointer', '$123.45')">
+          </figure>
+          <h3>Lucas</h3>
+          <p><span class="dogs">Cost to Adopt:</span> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+          <button  class="dogs-button" type="submit" onclick="adoptionFee(123.45)">Adopt</button>
+        </section>
+
+        <section class="intro-dogs">
+          <figure class="intro-img">
+            <img class="dogimage" src="images/jake-card.jpg" alt="Picture of Jake" onclick="dogInformation('Jake', 'labrador', '$123.45')">
+          </figure>
+          <h3>Jake</h3>
+          <p><span class="dogs">Cost to Adopt:</span> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+          <button  class="dogs-button" type="submit" onclick="adoptionFee(123.45)">Adopt</button>
+        </section>
+
+        <section class="intro-dogs">
+          <figure class="intro-img">
+            <img class="dogimage" src="images/angus-card.jpg" alt="Picture of Angus" onclick="dogInformation('Angus', 'Boxer', '$123.45')">
+          </figure>
+          <h3>Angus</h3>
+          <p><span class="dogs">Cost to Adopt:</span> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+          <button  class="dogs-button" type="submit" onclick="adoptionFee(123.45)">Adopt</button>
+        </section>
+
+        <section class="intro-dogs">
+          <figure class="intro-img">
+            <img class="dogimage" src="images/violet-card.jpg" alt="Picture of Violet" onclick="dogInformation('Violet', 'Labrador', '$123.45')">
+          </figure>
+          <h3>Violet</h3>
+          <p><span class="dogs">Cost to Adopt:</span> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+          <button  class="dogs-button" type="submit" onclick="adoptionFee(123.45)">Adopt</button>
```

Comment:
You do not need to include `type="submit"` this is just needed for buttons in forms